application/engine/services/autopilot_daemon.py

```
@@ -179,13 +180,134 @@ def _save_novel_state(self, novel: Novel) -> None:
+                self.context_builder.budget_allocator.theme_agent = agent
+
+    def _inject_skills_to_agent(self, agent, novel: Novel) -> None:
+        """根据 novel.enabled_theme_skills 将技能注入 Agent
+
+        通过覆盖 agent 的 _injected_skills 属性实现动态技能加载，
+        并 monkey-patch get_skills() 方法使其返回注入的技能列表。
+        同时加载用户自定义技能（custom_theme_skills 表）。
+        """
+        skill_keys = getattr(novel, 'enabled_theme_skills', []) or []
+        if not skill_keys:
+            # 无技能配置时，清除之前可能注入的技能
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)